trino/__init__.py

```diff
@@ -11,7 +11,28 @@
 # limitations under the License.
 
 from . import auth, client, constants, dbapi, exceptions, logging
+from ._version import (
+    __author__,
+    __author_email__,
+    __description__,
+    __license__,
+    __title__,
+    __url__,
+    __version__,
+)
 
-__all__ = ['auth', 'dbapi', 'client', 'constants', 'exceptions', 'logging']
-
-__version__ = "0.326.0"
+__all__ = [
+    "auth",
+    "client",
+    "constants",
+    "dbapi",
+    "exceptions",
+    "logging",
+    "__author__",
+    "__author_email__",
+    "__description__",
+    "__license__",
+    "__title__",
+    "__url__",
```

Comment:
can you remove these?

User's of the module generally should have no use for these properties.

Reply:
everything else other than `__version__`? It's kind of an old convention but it provides package related information after importing it. For examples when you run:

```python
import requests
help(requests)
```
gives the following output:

```
...

DATA
    __author_email__ = 'me@kennethreitz.org'
    __build__ = 141313
    __cake__ = '✨ 🍰 ✨'
    __copyright__ = 'Copyright 2022 Kenneth Reitz'
    __description__ = 'Python HTTP for Humans.'
    __license__ = 'Apache 2.0'
    __title__ = 'requests'
    __url__ = 'https://requests.readthedocs.io'
    chardet_version = '4.0.0'
    charset_normalizer_version = '2.1.1'
    codes = <lookup 'status_codes'>

VERSION
    2.28.1

AUTHOR
    Kenneth Reitz

...
```

It's not super essential but no harm keeping package information accessible with dunder.

Reply:
makes sense. I can see a few other packages following the convention.

Can you however extract all `setup.py` related preparatory changes (including this) to a separate commit?

Reply:
i.e. every change except in `test_client.py`, `client.py` and `constants.py` should be a preparatory commit.

It's safer to keep refactors separate from functional changes - helps pay better attention in reviews + easier to bisect or understand in future.

Reply:
I cannot make the change in `client.py` before refactoring that includes separating `__version__` from `trino/__init__.py`. Shall we merge the refactoring first?

Reply:
I created a refactoring PR https://github.com/trinodb/trino-python-client/pull/410 I can rebase this one after merging the refactoring.